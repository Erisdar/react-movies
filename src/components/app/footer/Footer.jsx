import BrandTitle from 'app/brandTitle/BrandTitle';
import React from 'react';
import './footer.css';

export default () => (
    <div className="footer app__footer">
        <BrandTitle className="footer__brand-title brand-title_color-red" />
    </div>
);