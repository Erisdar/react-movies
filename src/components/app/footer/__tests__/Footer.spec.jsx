import { shallow } from 'enzyme';
import React from 'react';
import Footer from '../Footer';

describe('Footer component', () => {
    it('should be render correctly', () => {
        const component = shallow(<Footer />);
        expect(component).toMatchSnapshot();
    });
});