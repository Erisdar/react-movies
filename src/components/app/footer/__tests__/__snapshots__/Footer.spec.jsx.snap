// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should be render correctly 1`] = `
<div
  className="footer app__footer"
>
  <_default
    className="footer__brand-title brand-title_color-red"
  />
</div>
`;
